packages/status-js/src/utils/predefined-bootstrap-nodes.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * source: @see https://github.com/waku-org/js-waku/blob/d2ac62c70809309c5052633482ccdff104f87ac8/src/lib/predefined_bootstrap_nodes.ts
+ * source: @see https://github.com/waku-org/js-waku/blob/d2ac62c70809309c5052633482ccdff104f87ac8/src/lib/random_subset.ts
+ */
+
+import { fleets } from '../consts/fleets'
+
+export const DefaultWantedNumber = 1
+
+export enum Fleet {
```

Comment:
I always prefer string union type over enums. The reason is that they don't have the equivalent in JavaScript and require runtime code. Also, they are easier to use because you don't need to import the type.

String union type will be completely removed during the build, whereas enums require code like this to work:
```js
var Direction;
(function (Direction) {
    Direction[Direction["Up"] = 0] = "Up";
    Direction[Direction["Down"] = 1] = "Down";
    Direction[Direction["Left"] = 2] = "Left";
    Direction[Direction["Right"] = 3] = "Right";
})(Direction || (Direction = {}));
```